doc/source/ray-overview/examples.rst

```diff
@@ -1173,7 +1186,7 @@ Ray Examples
         Simple Parallel Model Selection
 
     .. grid-item-card:: :bdg-secondary:`Code example`
-        :class-item: gallery-item core training
```

Comment:
I think we should remove this whole `ray-core/examples/plot_example-lm.html` example, the docs views are very low (also nowadays the `transformers` library is a better way to train these models than `fair-seq` :)

Reply:
Ack. Removing.